pyHS100/cli.py

```diff
@@ -165,14 +164,15 @@ def temperature(dev, temperature):
 @click.argument("h", type=click.IntRange(0, 360), default=None)
 @click.argument("s", type=click.IntRange(0, 100), default=None)
 @click.argument("v", type=click.IntRange(0, 100), default=None)
+@click.argument("period", type=click.IntRange(0, 10000), default=500)
```

Comment:
Is 10000 a limit enforced by the bulb? If yes, maybe it should also be enforced on SmartBulb class too?

Reply:
It seems like the bulb doesn't have any limit. I can remove this range from CLI

Reply:
There must be some kind of limitation, but ok!